src/main/scala/com/fulcrumgenomics/bam/ClipBam.scala

```diff
@@ -55,7 +55,7 @@ import scala.collection.immutable.IndexedSeq
     |done in streaming fashion with, for example:
     |
     |```
-    |samtools sort -n -u in.bam | fgbio ClipBam -i /dev/stdin ...
+    |fgbio --compression 0 SortBam -i in.bam -o out.bam -s queryname | fgbio ClipBam -i /dev/stdin ...
```

Comment:
Can go back to samtools since beecb49f2b3472d2e8589460b06c7f9753781ef2 (here and elsewhere).